sources/packages/api/src/route-controllers/supporting-user/supporting-user.aest.controller.ts

```diff
@@ -39,7 +37,7 @@ export class AESTSupportingUserController {
         ({
           supportingUserId: supportingUser.id,
           supportingUserType: supportingUser.supportingUserType,
-        } as ApplicationSupportingUsersDTO),
+        } as ApplicationSupportingUsersApiOutDTO),
```

Comment:
Can you please remove the cast here due to the reasons that we talked about in the last meeting?